File: source/code/csharp/quickstart/authentication_attempt.cs
try
{
     var loginResult = await myApp.AuthenticateAccountAsync(
        "tk421@galacticempire.co", "Changeme123!");
     var loggedInAccount = await loginResult.GetAccountAsync();

     Console.WriteLine("User {0} logged in.", loggedInAccount.FullName);
}
catch (ResourceException rex)
{
    Console.WriteLine("Could not log in. Error: " + rex.Message);
}

File: source/code/python/quickstart/authentication_attempt.py
from stormpath.error import Error as StormpathError

try:
    result = application.authenticate_account('tk421@galacticempire.co', 'Changeme123!')
    account = result.account

    print('User ' + result.account.full_name + ' logged in.')
except StormpathError, e:
    print('Could not log in. Error: ' + e.user_message)
